### PR TITLE
Bump requirements & update the README

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,20 +7,14 @@
...
-## Installation
-You may install the markdown-xblock using its setup.py, or if you prefer to use pip, run:
+## Installation with [Tutor](https://docs.tutor.edly.io)
+You may install the markdown-xblock to your Tutor environment by adding it to the `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
 
-```shell
-pip install markdown-xblock==1.3.0
 ```
-
-If you prefer to install directly from Git, run:
```
Comment (fghaas) [inline README.md:17]: Perhaps we want to retain this information? We could mention that `OPENEDX_EXTRA_PIP_REQUIREMENTS` also supports `git+https` entries (unless it actually does not, which would be news to me).

File: README.md [hunk 2]
```diff
@@ -7,20 +7,14 @@
...
-
-```shell
-pip install git+https://github.com/citynetwork/markdown-xblock.git@v1.3.0
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
+- markdown-xblock==1.3.0
 ```
-
-You may specify the `-e` flag if you intend to develop on the repo.
+For additional information, please refer to the official [documentation](https://docs.tutor.edly.io/configuration.html#installing-extra-xblocks-and-requirements).
```
Comment (fghaas) [inline README.md:17]: I may be missing something, but to me it looks like the `installing-extra-xblocks-and-requirements` anchor does not exist on that page.

**Review by fghaas — COMMENTED** [review]
The requirements change looks fine to me; I have a couple of questions about the README.

File: README.md (hunk 1 above)
Comment (mrtmm) [reply README.md:17]: Sure, I added it back.

File: README.md (hunk 2 above)
Comment (mrtmm) [reply README.md]: You are right, I updated the link.

**Review by fghaas — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**